### DISCUSSION TITLE
Help to setup the bot

### DISCUSSION THREAD

**Discussion by Ib69** [GilbertGobbels/GAwesomeBot#768] [Q&A]
Hello...
Can somebody help me to setup the bot, I need to make it online... Bruh
its not working, sorry for that...
HELP PLZZZ !!

**Comment by KnowZbox** [comment]
no

**Comment by Vault108** [comment]
> Hello...
> Can somebody help me to setup the bot, I need to make it online... Bruh
> its not working, sorry for that...
> HELP PLZZZ !!

Are you getting any sort of error message
What have you tried?

**Reply by B33Man** [reply]
I am would you mind helping me out

**Reply by BRAVO68WEB** [reply]
I can help

**Reply by B33Man** [reply]
Ok dm me at Bee_Man#7545
